### PR TITLE
feat!: Change input structure

### PR DESCRIPTION
**Pull request by Anton-L-Ext**

### CODE REVIEW

File: examples/main.tf [hunk 1]
```diff
@@ -1,6 +1,17 @@
-module "slo" {
-  source       = "../"
-  service_name = "Demo Service"
-  project      = "hiiretail-monitoring-prod-6500"
-  slos         = yamldecode(file("slos.yaml"))
+module "services-and-slos" {
```
Comment (SnigL) [inline examples/main.tf]: No general guidelines that module names should be all underscores or something?

File: examples/main.tf [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+  source  = "../"
+  project = "my-gcp-project"
+  services    = yamldecode(file("services-and-slos.yaml"))
+  default_user_labels = {
+    set_this_on_all_resources = true
+  }
+
+  # Notificaiton channels: Alt 1 - List of notification channels ids
```
Comment (SnigL) [inline examples/main.tf]: Notificaiton > Notification

File: README.md [hunk 3]
```diff
@@ -1,36 +1,135 @@
...
-| __fallback_notification_channels__ | NCs to be set for all alerts that don't provide `notification_channels`. Provide the NCs "id" or "display name" (the latter is dependant on the notification_channel_ids variable) | `list(any)` | []      |    no    |
-| __notification_channel_ids__       | To be able to provide channels display name instead of id/name, provide a  be { display_name: name } or output from tf-module-gcp-notification-channels.                           | `list(any)` | []      |    no    |
-| __default_alert_documentation__    | Documentation to be set for all burn-rate alerts.                                                                                                                                  | `string`    | n/a     |    no    |
-| __slos__                           | The list of alert policies configurations.                                                                                                                                         | `list(any)` | n/a     |   yes    |
+For creating SLOs in GCP Monitoring.
 
-## Burn rate alerting
```
Comment (SnigL) [inline README.md:14]: I see code about burn rate alerts but nothing described in the docs. Are these alerts still created when you setup the SLO?

Comment (Anton-L-Ext) [reply README.md:14]: Yes, it creates one burn-rate alert (fast-burn). I've updated the readme, take a look and let me know if its good enough ^^

**Review by SnigL — APPROVED** [review]

**Pull request merged**